src/SSHLibrary/pythonclient.py

```diff
@@ -30,7 +30,10 @@
 # There doesn't seem to be a simpler way to increase banner timeout
 def _custom_start_client(self, event=None, timeout=None):
     self.banner_timeout = 45
-    self._orig_start_client(event, timeout=timeout)
+    if timeout is not None:
+        self._orig_start_client(event, timeout=timeout)
+    else:
+        self._orig_start_client(event)
 
```

Comment:
I think you could do better without the condition by passing the keyword arguments to the other function:
```python
def _custom_start_client(self, event=None, **kwargs):
      self.banner_timeout = 45
      self._orig_start_client(event, **kwargs)		 
```
This will also be compatible with whatever arguments are added in the future.

Actually, is the signature of _custom_start_client and _orig_start_client seem to be the same. Then I suppose it should be
```python
def _custom_start_client(self, *args, **kwargs):
      self.banner_timeout = 45
      self._orig_start_client(*args, **kwargs)		 
```